src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        String[] machines = new String[] {"Truck", "Bulldozer", "Excavator"};
+        for (String machine : machines) {
```

Comment:
why strings? they should be new objects of your classes
(`new Truck()`, `new Bulldozer()`)